File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.py
class Solution:
    def minMoves2(self, nums: List[int]) -> int:
        nums.sort()
        left = 0
        right = len(nums)-1
        mid = floor(len(nums)/2)
        count = 0
        while (left != mid or right != mid):
            if left != mid:
                count += nums[mid]-nums[left]
                left += 1
            if right != mid:
                count += nums[right]-nums[mid]
                right -= 1
            
        return count

File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.kt
class Solution {
    fun minMoves2(nums: IntArray): Int {
        nums.sort()
        var left = 0
        var right = nums.size-1
        var mid = nums.size /2
        var count = 0
        while (left != mid || right != mid){
            if (left != mid) {count += nums[mid]-nums[left++]}
            if (right != mid) {count += nums[right--]-nums[mid]}
        }
        return count
    }
}